src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Utilities/DeviceTest.cs

```diff
@@ -13,11 +13,14 @@ namespace Xamarin.Android.Build.Tests
 {
 	public class DeviceTest: BaseTest
 	{
+		public static string GuestUserName = "guest1";
```

Comment:
Shouldn't this be `const` or `readonly`?  I can't imagine any reason to need to be able to *change* this value.